eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshServer.java

```diff
@@ -73,14 +73,16 @@ public EventMeshServer() {
 
         final List<String> provideServerProtocols = configuration.getEventMeshProvideServerProtocols();
         for (final String provideServerProtocol : provideServerProtocols) {
-            if (ConfigurationContextUtil.HTTP.equals(provideServerProtocol)) {
-                BOOTSTRAP_LIST.add(new EventMeshHttpBootstrap(this));
-            }
-            if (ConfigurationContextUtil.TCP.equals(provideServerProtocol)) {
-                BOOTSTRAP_LIST.add(new EventMeshTcpBootstrap(this));
-            }
-            if (ConfigurationContextUtil.GRPC.equals(provideServerProtocol)) {
-                BOOTSTRAP_LIST.add(new EventMeshGrpcBootstrap(this));
+            switch (provideServerProtocol) {
```

Comment:
if you change to switch, you need add default case.

Reply:
> if you change to switch, you need add default case.

I hava fix it and resubmit again

Reply:
you need add default in switch condition. I think you just need to add:
switch (condition) {
 case 1:
 case 2:
default:
    break;
}